usaspending_api/transactions/delta_models/published_fabs.py

```diff
@@ -0,0 +1,120 @@
+PUBLISHED_FABS_COLUMNS = {
+    "published_fabs_id": {"delta": "INTEGER", "postgres": "UUID"},
+    "afa_generated_unique": {"delta": "STRING", "postgres": "TEXT"},
+    "action_date": {"delta": "STRING", "postgres": "TEXT"},
+    "action_type": {"delta": "STRING", "postgres": "TEXT"},
+    "action_type_description": {"delta": "STRING", "postgres": "TEXT"},
+    "assistance_type": {"delta": "STRING", "postgres": "TEXT"},
+    "assistance_type_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "award_description": {"delta": "STRING", "postgres": "TEXT"},
+    "award_modification_amendme": {"delta": "STRING", "postgres": "TEXT"},
+    "awardee_or_recipient_legal": {"delta": "STRING", "postgres": "TEXT"},
+    "awardee_or_recipient_uniqu": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_agency_code": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_agency_name": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_office_code": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_office_name": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_sub_tier_agency_c": {"delta": "STRING", "postgres": "TEXT"},
+    "awarding_sub_tier_agency_n": {"delta": "STRING", "postgres": "TEXT"},
+    "business_categories": {"delta": "ARRAY<STRING>", "postgres": "ARRAY[]TEXT"},
```

Comment:
I believe the Postgres type here should be `TEXT[]`

Reply:
addressed